cli/Valet/PhpFpm.php

```diff
@@ -192,8 +258,31 @@ public function useVersion($version, $force = false)
             $this->brew->ensureInstalled($version, [], $this->taps);
         }
 
-        // Unlink the current php if there is one
+        if ($directory) {
+            $oldCustomPhpVersion = $this->site->customPhpVersion($site); // Example output: "74"
+            $this->cli->quietly('sudo rm '.VALET_HOME_PATH.'/'.$this->fpmSockName($version));
```

Comment:
No actually, we are deleting `.sock` files before making a switch to prevent this issue. https://github.com/laravel/valet/commit/dd17bb9e9add5e7b660337f395455e96193e2684

Reply:
Hm.. should this be deleting the sock file for the `old` version then, not the new one? I guess I don't see the value of deleting the sock file for the version we're *switching* to.